### ISSUE TITLE
Prep for Removal of instance-group-level and global `properties`

### ISSUE THREAD

**Issue by jhunt** [genesis-community/jumpbox-genesis-kit#24]
As of BOSH 270.5.0, BOSH is done with v1 manifest stuff. We need to migrate all of the global and instance-group-level properties to job-level properties.

**Comment by jhunt** [comment]
Here's a start:

```
→  find manifests/ -name '*.yml' | xargs -n1 spruce json | jq -r '.properties'
null
null
{
  "jumpbox": {
    "env": {
      ".": "(( inject params.env_vars ))",
      "TMPDIR": "/var/vcap/data/root_tmp"
    },
    "hostname": "(( grab params.hostname ))",
    "users": "(( grab params.users ))"
  }
}
```

**Comment by dennisjbell** [comment]
Fixed in 96c2e7c3cbadc03c134735fa22f9a7a062890027
